tests/client/amo/containers/TestApp.js

```diff
@@ -35,59 +37,50 @@ describe('App', () => {
 
   const FakeInfoDialogComponent = () => <div />;
 
-  it('renders its children', () => {
-    // eslint-disable-next-line react/no-multi-comp
-    class MyComponent extends React.Component {
-      render() {
-        return <p>The component</p>;
-      }
-    }
-    const i18n = getFakeI18nInst();
-    const location = sinon.stub();
-    const root = renderIntoDocument(
-      <AppBase i18n={i18n} isAuthenticated
+  function render({ children = [], ...customProps } = {}) {
```

Comment:
Hah, thanks for this, I saw it when I was working on #1253 and thought "we should put this in a render function". 👍